src/screens/LoginWithPinScreen.tsx

```diff
@@ -58,9 +60,20 @@ const LoginWithPinScreen = (props: LoginWithPinScreenProps) => {
 
   return (
     <Screen contrastedBg {...props}>
-      {isPinModalVisible && <ConfirmWithAuthModal usePin onConfirm={handleSuccessfulLogin} />}
+      <AuthenticationModal visible={isPinModalVisible} forcePinUsage onConfirm={handleSuccessfulLogin} />
+      {!isPinModalVisible && (
+        <Message>
+          <AppText>Nice 👍 Unlocking...</AppText>
+        </Message>
+      )}
```

Comment:
This will be shown while we are loading some initial data when the app is first launched. Then it will disappear much quicker after data has been initialized.

Reply:
What about showing the loading spinner instead? Just a bit less confusing IMO as you don't have to read anything, you understand that you just need to wait.

Reply:
Sure. My initial thought is that by having some text it'd be easier to debug in case someone gets stuck on this screen for whatever reason. I'll leave the word "Unlocking" with a spinner, if you don't mind.

Reply:
Done ✅